src/requests.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+// todo, add catch and error handling
+
+const gamesApi = axios.create({
+    baseURL: 'https://shadia-nc-games.herokuapp.com/api',
+});
+
+export const getReviews = () => {
+    return gamesApi
+        .get('/reviews')
+        .then((res) => {
+            return res;
+        })
+};
+
+export const getCategories = () => {
+    return gamesApi
+        .get('/categories')
+        .then((res) => {
+            return res;
+    })
+}
+
+export const getCategoryReviews = (category) => {
+    return gamesApi
+        .get('/reviews', { params: { category } })
+        .then((res) => {
+            return res;
+        })
+};
```

Comment:
To make it a bit drier - this function and lines 9-15 are the same thing, you can use the same function. 

With the params object, if you pass it a category, it will add the query e.g. ?category=catHere
if you dont pass it a category/they are undefined, it will just do the normal request, like in lines 9 -15